src/utils/constants.ts

```diff
@@ -0,0 +1,25 @@
+export enum Market {
+  BaseAsset = '5ac9f65c0efcc4775e0baec4ec03abdde22473cd3cf33c0419ca290e0751b225',
+  QuoteAsset = '9b3d7db7f93cae59e9407232b9864b54c21cba20b992d707fa45136fbfa9b8fa',
+}
+
+export const Markets: any = {
+  markets: [
+    {
+      market: {
```

Comment:
This is the third place where you harcode the assets.

Reply:
fixed this